AdblockPlusSafari/AdblockPlusController.m

```diff
@@ -100,6 +115,21 @@ -(UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NS
   return cell;
 }
 
+- (NSString *)tableView:(UITableView *)tableView titleForFooterInSection:(NSInteger)section
```

Comment:
This is one method from UITableViewDelegate. It is responsible for displaying footer section. In case of ABP the footer is "Last filter list update: Friday, Octover 16, 2015..."